templates/bare.mustache

```diff
@@ -11,5 +11,5 @@ SECTION "main", ROM0[$0150]
 
 main:
 {{#body}}
-{{.}}
+{{{.}}}
```

Comment:
this stops html entities being escaped in the output, which we don't want because we use quotation marks in file inclusion that would be rendered as &quot;